verify/rpc-api-v17.txt

```diff
@@ -18,7 +18,7 @@ gettxout "txid" n ( include_mempool )
 gettxoutproof ["txid",...] ( blockhash )
 gettxoutsetinfo
 preciousblock "blockhash"
-pruneblockchain
+pruneblockchain height
```

Comment:
These file was created using `bt17 help > rpc-api-v17.txt`. It should not need updating.